micromamba/tests/test_create.py

```diff
@@ -1231,9 +1231,8 @@ def test_create_from_oci_mirrored_channels(tmp_home, tmp_root_prefix, tmp_path,
     assert pkg["name"] == "pandoc"
     if spec == "pandoc=3.1.13":
         assert pkg["version"] == "3.1.13"
-    assert pkg["base_url"].startswith(
-        "https://pkg-containers.githubusercontent.com/ghcr1/blobs/pandoc"
-    )
+    assert pkg["base_url"] == "https://pkg-containers.githubusercontent.com/ghcr1/blobs"
+    assert pkg["channel"] == "https://pkg-containers.githubusercontent.com/ghcr1/blobs"
```

Comment:
We could set this to something else like `oci` or `ghcr`?
But that could be done in another PR once it's all sorted out.

Reply:
Yeah let's fix the bugs for now;, and figure out what to do for OCI in a dedicated PR.